### DISCUSSION TITLE
Home IdP Discovery config Empty (20.0.0)

### DISCUSSION THREAD

**Discussion by claudiuproject** [sventorben/keycloak-home-idp-discovery#191] [Q&A]
Hi,

I downloaded the jar for version 20.0.0 but when I access the module, I don't have the options inside.

I access the VM using port fw, and the machine has an private ip, is possible this can be an possible reason?

If you can help me, will be great.

Thx!

![image](https://user-images.githubusercontent.com/83223514/234303687-27aac6be-dc82-48a4-8fdd-8040b9db0dea.png)

docker compose:
![image](https://user-images.githubusercontent.com/83223514/234303879-8fc448dc-ad9b-438f-a4a0-87ee76e617db.png)

dockerfile:
![image](https://user-images.githubusercontent.com/83223514/234303958-51576dae-1741-4365-aadd-14b195df54be.png)

**Comment by sventorben** [comment]
Please try a newer Keycloak version. There were several issues in Keycloak 20.0.0 around configuration of custom authenticators.
